code/game/turfs/simulated/floor_icon.dm

```diff
@@ -308,7 +308,7 @@ var/list/flooring_cache = list()
 	if(!flooring_cache[cache_key])
 		var/image/I = image(icon = 'icons/turf/damage_overlays.dmi', icon_state = icon_base)
 
-		I.plane = FLOOR_PLANE
```

Comment:
when would a floor not be on the floor plane?

Reply:
Floor plane is no longer constant, but decal still needs to be be on the same plane as the floor